src/integrations/admin/addon-installation/dialog-integration.php

```diff
@@ -4,6 +4,7 @@
 
 namespace Yoast\WP\SEO\Integrations\Admin\Addon_Installation;
 
+use WPSEO_Admin_Utils;
```

Comment:
It's not clear to me why this import is added.

Reply:
Automatically added when I thought I needed a util function, doesn't automatically remove it when it turned out I didn't. removed it.